### ISSUE TITLE
[A11Y] Android: Sidebar Menu and close buttons are not labeled

### ISSUE THREAD

**Issue by gkhadke** [ampproject/ampstart#373]
**ENVIRONMENT:**

OS (number): N (7.1.2)
Build number: N2G47H
Form factor (Smartphone / tablet):  Nexus 6P
Browser: Chrome Dev 59.0.3056.4

**Pre-requisite:**
1. Talkback is ON

**STEPS TO REPRODUCE:**

1. Open [https://ampstart.com](https://ampstart.com)))
2. Try to Navigate to Sidebar Menu and content within it.

**ACTUAL RESULTS:**

1. When user navigates to the Menu button, talkback reads it “button, double tap to activate”. It doesn’t reads out any label for it.
2. Same issue is observed when user navigates on the close (X) button present in the sidebar Menu. Screenreader user is also provided with an invisible close Menu button additionally, which can be reached at the end after navigating through the Menu items

**EXPECTED RESULTS:**

1. Sidebar Menu button should be labeled appropriately.
2. There should be only one close button  and it should have appropriate label.

**ATTACHMENTS**

<img width="1440" alt="unlabeledsidebarmenu_button" src="https://cloud.githubusercontent.com/assets/26986019/24814291/ee4b4fc4-1b85-11e7-8e8f-f40e25b2c50d.png">
<img width="1440" alt="unlabeled close button" src="https://cloud.githubusercontent.com/assets/26986019/24814322/08c18378-1b86-11e7-9212-e383f8331183.png">

**Comment by torch2424** [comment]
Fixed in my a11y bugs branch

[Commit Fix](https://github.com/torch2424/ampstart/commit/eeb8d43395d09cb39e26ac860625acbbcd310507)

**Comment by camelburrito** [comment]
Always send Prs :)
